samples/unit-tests/maps/mapbubble-linewidth/demo.js

```diff
@@ -56,7 +56,41 @@ QUnit.test('MapBubble', function (assert) {
     assert.strictEqual(
         chart.series[1].joinBy[0],
         'hc-key',
-        `The value of joinBy in the mapbubble series 
+        `The value of joinBy in the mapbubble series
         should be 'hc-key' by default.`
     );
 });
+
+QUnit.test('AddPoint', function (assert) {
```

Comment:
To avoid having to create another chart instance, can we instead do the `addPoint` test on the other chart in the file? In order to keep `gulp test` time down we generally try to reuse instances whenever it makes sense.